lib/gcloud/vision/image.rb

```diff
@@ -359,8 +356,9 @@ def inspect
       ##
       # @private The Google API Client object for the Image.
       def to_gapi
-        if content?
-          { content: content }
+        if io?
+          @io.rewind
+          { content: Base64.strict_encode64(@io.read) }
```

Comment:
Would it be safer for memory usage (see #718) to [do this instead](https://github.com/GoogleCloudPlatform/gcloud-ruby/commit/19c47947b22dc0a09c9ac0a2a8ed30a7b2cbdbc3)?